apps/graphql/package.json

```diff
@@ -30,7 +30,8 @@
     "serverless-offline": "^4.9.4",
     "serverless-webpack": "^5.2.0",
     "uglifyjs-webpack-plugin": "^2.1.2",
-    "webpack": "^4.29.1"
+    "webpack": "^4.29.1",
+    "webpack-node-externals": "^1.7.2"
```

Comment:
This is not really related, but okay :D

Reply:
Maybe not, but it still gave eslint error